app/controllers/override/registrations_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Override
```

Comment:
I think it would be more readable if you name this module as `User` than `Override`. We would know what this module "overrides" :)

Reply:
I think this is taken directly from docs of `devise_token_auth`: https://devise-token-auth.gitbook.io/devise-token-auth/usage/overrides#custom-controller-overrides

And I don't have a problem with that - putting it in `User` may on other hand make it look like "our" code, where in reality this is just overriding existing defaults from some gem. So there is not really a better solution between those two, both are valid. And I would probably stick to documented way, so someone that just took a look at documentation of the gem will be able to find this override easily.